.github/workflows/deploy_EC2.yaml

```diff
@@ -54,6 +54,8 @@ jobs:
           # Turn on Route53 DNS
           # Requires configuring R53 in your AWS account.
           aws_r53_enable: true
-          domain_name: bitovi-sandbox.com
-          sub_domain: deno-ec2
+          aws_r53_domain_name: bitovi-sandbox.com
+          aws_r53_sub_domain_name: deno-ec2
           aws_r53_enable_cert: true
```

Comment:
I was going comment that in the future, when we have cloud-agnostic actions, won't it make more sense to have agnostic variable names; as I was writing it, I thought that there might still be reasons to have different names across the different clouds, and when that day comes, we can just set `aws_x` to `x` if set.